semgrep_output_v1.atd

```diff
@@ -885,7 +885,9 @@ type ecosystem <python decorator="dataclass(frozen=True)"> <ocaml attr="deriving
   | Nuget <json name="nuget">
   | Pub <json name="pub">
   | SwiftPM <json name="swiftpm">
+  (* Deprecated: Mix is a build system, should use Hex, which is the ecosystem *)
```

Comment:
Can we juste delete Mix instead of adding the deprecated notice?
Is there any rules/code relying on Mix?